data/legend/metadata/hardware/configuration/channelmaps/validity.yaml

```
@@ -0,0 +1,21 @@
+- valid_from: 20221128T000000Z
+  apply:
+    - l200-p02-r%-T%-all-config.json
+    - l200-p02-r006-T%-all-config.json
+- valid_from: 20221128T000000Z
+  category: cal
+  apply:
+    - l200-p02-r%-T%-all-config.json
+    - l200-p02-r%-T%-cal-config.json
+    - l200-p02-r006-T%-all-config.json
+- valid_from: 20230117T000000Z
+  mode: replace
+  apply:
+    - l200-p02-r006-T%-all-config.json
+    - l200-p02-r009-T%-all-config.json
+- valid_from: 20230117T000000Z
+  mode: replace
+  category: cal
+  apply:
+    - l200-p02-r006-T%-all-config.json
+    - l200-p02-r009-T%-all-config.json
```

Our code breaks here:
```suggestion
- valid_from: 20221128T000000Z
  apply:
    - l200-p02-r%-T%-all-config.yaml
    - l200-p02-r006-T%-all-config.yaml
- valid_from: 20221128T000000Z
  category: cal
  apply:
    - l200-p02-r%-T%-all-config.yaml
    - l200-p02-r%-T%-cal-config.yaml
    - l200-p02-r006-T%-all-config.yaml
- valid_from: 20230117T000000Z
  mode: replace
  apply:
    - l200-p02-r006-T%-all-config.yaml
    - l200-p02-r009-T%-all-config.yaml
- valid_from: 20230117T000000Z
  mode: replace
  category: cal
  apply:
    - l200-p02-r006-T%-all-config.yaml
    - l200-p02-r009-T%-all-config.yaml
```